releases.md

```diff
@@ -0,0 +1,51 @@
+# Release Process
+
+This process is a work-in-progress until the first major release of the project.
+
+The process will utimately aim to have:
+
+- A published release date for the first caikit major release, upon the availability of the first beta release
+- A well-known cyclical release dates for caikit minor and patch releases, upon the availability of the first caikit major release
+- Minor releases backwards compatible with respect to the API, upon the availability of the first caikit major release
+
+## Semantic versioning
+
+caikit versions are expressed as x.y.z, where x is the major version, y is the minor version, and z is the patch version, following [Semantic Versioning](https://semver.org/spec/v2.0.0.html) terminology.
+
+## Patch releases
+
+Patch releases provide users with bug fixes and security fixes. They do not contain new features.
+
+Patch releases should be done every second week on a Wednesday (assuming there are changes since the last release). A patch release to fix a high priority regression or security issue does not have to follow this schedule, but it is highly desirable that it is released on a Wednesday if possible.
+
+### Cancelling a patch release
+
+A patch release should be cancelled:
+
+- If it falls within the week of a minor release
+
+## Minor releases
+
+Minor releases contain security and bug fixes as well as new features.
+
+**Note:** They will be backwards compatible with respect to the API, upon the availability of the first caikit major release. Until then, they will break the API.
+
+Minor releases should be done every other week inbetween path releases on a Wednesday (assuming there are changes since the last release).
```

Comment:
Both patch and minor (feature) releases would be regular. Patch would be more often but your minor can be scheduled to what suits the project. This will depend on rate of change in the project.

Scheduled release would be post a stable/major release.